packages/react-components/src/types/Attachment.ts

```diff
@@ -4,13 +4,16 @@
 import { ChatMessage } from './ChatMessage';
 import { AttachmentMetadata } from '@internal/acs-ui-common';
 
+/* @conditional-compile-remove(file-sharing-acs) */
 /**
  * @beta
  *
  * Attachment Options that defines behaviour for uploading and downloading attachments.
  */
 export interface AttachmentOptions {
+  /* @conditional-compile-remove(file-sharing-acs) */
   uploadOptions?: AttachmentUploadOptions;
+  /* @conditional-compile-remove(file-sharing-acs) */
   downloadOptions?: AttachmentDownloadOptions;
```

Comment:
can we just cc flag for the whole thing and not individually for upload/download options?

Reply:
thanks updated